api/api/templates/admin/api/media_report/change_form.html

```diff
@@ -0,0 +1,159 @@
+{% extends "admin/change_form.html" %}
+{% load static %}
+
+{% block extrahead %}
+{{ block.super }}
+<style>
+  audio, .img-container {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+  .img-container {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+  }
+  .img-container img, .img-container p {
+    grid-column: 1 / -1;
+    grid-row: 1 / -1;
+  }
+  .img-container p {
+    border-radius: 4px;
+    padding: 10px;
+    z-index:1;
+    pointer-events: none;
+    color: black;
+    background: rgba(255, 255, 255, 0.5);
+    text-align: center;
+    font-weight: bold;
+  }
+  .blur-image {
+    transition: filter 0.3s;
+    filter: blur(60px);
+  }
+  .no-blur {
+    filter: none;
+  }
+  .no-blur ~ p {
+    display: none;
+  }
+</style>
+{% endblock %}
+
+{% block after_field_sets %}
+{{ block.super }}
+
+{% if not add %}
+<fieldset class="module aligned">
+  <h2>Media details</h2>
+  {% if media_type == 'image' %}
+    <div class="img-container">
+      <img src="{% url media_type|add:'-thumb' identifier=identifier %}" alt="Media Image" class="blur-image" height="300"
+           onclick="toggleBlur(this)"
+           onerror="this.onerror=null;this.src='{{ url }}';"
+           style="cursor: pointer;">
+      <p>Show content</p>
+    </div>
+
+  {% elif media_type == 'audio' %}
+    <audio controls>
+      <source src="{{ url }}" type="audio/mpeg">
+      Your browser does not support the audio element.
+    </audio>
+  {% endif %}
+
+  <div class="form-row field-created_at">
+      <div class="flex-container">
+        <label>Title and creator</label>
+        <div class="readonly">
+          <p>{{ title }} {% if creator_url %}by <a href="{{creator_url}}">{{creator}}</a> {% else %} {{creator}} {% endif %} </p></div>
```

Comment:
Can we lean on the attribution generation here to make things easier? It already resolves the creator_url/creator resolution, printing the title, source, etc.

Reply:
Apparently, we don't have a function to get the html attribution, only the text one that does not add creator and work urls.
For some reason, I was under the impression that all the items mentioned in the IP need to be on separate lines. Now, I updated the form to display the media details more compactly. For attribution, I created a new sub-template.